docs/fuzzing.md

```diff
@@ -77,72 +72,30 @@ Then, start fuzzing! AFL and AddressSanitizer have a lot of settings, so please
 for the details. The sample way to run fuzzer is like below:
 
 ```
-CPU=0
-FUZZER=/path/to/AFLplusplus/afl-fuzz
-
-sudo AFL_NO_AFFINITY=1 \
-     ASAN_OPTIONS=detect_leaks=0:abort_on_error=1:symbolize=0 \
-     BPFTRACE_MAX_AST_NODES=200 \
-     taskset -c ${CPU} \
-     $FUZZER -M 0 -m none -i ./input -o ./output -t 3000 -- \
-     ./src/bpftrace_fuzz @@
+AFL_NO_AFFINITY=1 \
+ASAN_OPTIONS=abort_on_error=1,symbolize=0 \
+BPFTRACE_BTF= \
+BPFTRACE_MAX_AST_NODES=200 \
+BPFTRACE_AVAILABLE_FUNCTIONS_TEST= \
+afl-fuzz -a text -M 0 -m none -i ./input -o ./output -t 3000 -- \
+     src/bpftrace --test=codegen @@ 2>/dev/null
 ```
 
 I describe several important things:
 
-- `bpftrace_fuzz` is a fuzzer that we built. It's a slightly modified version of bpftrace for fuzzing, and
-  the first argument is the script file name. If the argument is not given, it reads a script from stdin.
 - `-i` is the input directory, and `-o` is the output directory. In the input directory, you need to put
   something to start fuzzing. The most simple example is `echo a > input/a`. More sophisticated inputs can
   be created by extracting the bpftrace program from the source code directory (especially from tests
   directory. See the "Input corpus creation" section below for the details.) If some inputs that cause a
   program crash is found, `output/crashes` contains them.
-- bpftrace has known that it has several memory leaks. Therefore `ASAN_OPTIONS=detect_leaks=0` is needed.
-  Otherwise, fuzzer thinks each memory leak as a crash and report it. Also,
-  `abort_on_error=1: symbolize=0` is required for fuzzing.
+- bpftrace has known that it has several memory leaks. Therefore `ASAN_OPTIONS=detect_leaks=0` may also needed.
```

Comment:
I've plugged all the leaks. Any new leaks are push blocking IMO

Reply:
Dropped.